ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/util/BeaconStateUtil.java

```diff
@@ -133,9 +132,12 @@ public static BeaconState get_initial_beacon_state(
     }
 
     // Process initial activations
+    int index = 0;
     for (Validator validator : state.getValidator_registry()) {
-      if (validator.get_effective_balance().compareTo(UnsignedLong.valueOf(MAX_DEPOSIT_AMOUNT))
-          >= 0) activate_validator(state, validator, true);
+      List<UnsignedLong> balances = state.getValidator_balances();
+      if (balances.get(index).compareTo(UnsignedLong.valueOf(MAX_DEPOSIT_AMOUNT)) >= 0)
```

Comment:
Just because I'm picky, I usually err on the side of using braces even if they're not necessary. Your call.

Reply:
ya i do too, but this code was like that before.  i can update it